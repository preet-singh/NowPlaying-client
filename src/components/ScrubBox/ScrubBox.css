@media screen and (min-width: 320px){

    .scrub-box {
    display: flex;
    flex-direction: column-reverse;
    width: 100%;
  }

}

@media screen and (min-width: 768px){

  .scrub-box {
    height: 100px;
    padding: 10px;
    padding-left: 50px;
    text-align: left;
    margin-right: auto;
    width: 40%;
  }

  .scrub-box button {
    
  }
}