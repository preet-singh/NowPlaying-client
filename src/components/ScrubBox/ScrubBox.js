import React from 'react';
import './ScrubBox.css';
import PlayButton from '../PlayButton/PlayButton';
import ScrubBar from '../ScrubBar/ScrubBar';

function ScrubBox(props) {
  return (
    <div className='scrub-box'>
      <PlayButton />
      <ScrubBar />
    </div>

  )
}

export default ScrubBox;