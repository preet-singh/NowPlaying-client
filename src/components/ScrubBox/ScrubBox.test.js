import React from 'react';
import ScrubBox from './ScrubBox.js';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`ScrubBox component`, () => {

  it('renders the ScrubBox', () => {
    const wrapper = shallow(<ScrubBox />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})