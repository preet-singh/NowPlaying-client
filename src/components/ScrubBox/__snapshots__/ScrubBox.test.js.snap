// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrubBox component renders the ScrubBox 1`] = `
<div
  className="scrub-box"
>
  <PlayButton />
  <ScrubBar />
</div>
`;
